Library/Formula/git-vendor.rb

```diff
@@ -0,0 +1,15 @@
+class GitVendor < Formula
+  desc "command for managing git vendored dependencies"
+  homepage "https://brettlangdon.github.io/git-vendor"
+  url "https://github.com/brettlangdon/git-vendor/archive/v1.0.0.tar.gz"
```

Comment:
I actually just pushed an updated version, so if we could make this `v1.1.0` that would be greatly appreciated :)